app/assets/stylesheets/config/_colors.css

```diff
@@ -0,0 +1,23 @@
+html {
+  --light-beige: #E8DBB3;
+  --yellow: #FFD96A;
+  --red: #D92A45;
+  --light-blue: #7387EB;
+  --blue: #2347EB;
```

Comment:
**suggestion:** we should move these to `:root` in `_base`. WDYT? 

I also like to prefix variables with their type, so I'd do something like `--color--light-beiger`. But I'm fine with not doing that.

Reply:
I moved before I saw this, so, yes, completely agree 😂